Enabled and configured filters

dmcarmo opened the pull request:
Re-enabled the filters on the organisations page, cleaned up the corresponding code and made some style changes.

dmcarmo committed:
Enabled and configured filters

arieivs commented on app/assets/stylesheets/components/_filter.scss:
```diff
@@ -0,0 +1,31 @@
+.simple_form {
```
hmmm isto não é um bom nome 😅 eu diria ou algo genérico mas representativo daquilo que é, tipo "w-48" (e aí acho que mudava esta regra para o layout.css) ou então um nome específico que só uses aqui tipo "organisations-filter-field"
Diria que a primeira opção é melhor, mas fica à escolha

arieivs commented on app/assets/stylesheets/components/_index.scss:
```diff
@@ -5,6 +5,7 @@
 @import "blockquote";
 @import "button";
 @import "card";
+@import "filter";
```
eu acho que me vou baralhar no futuro entre os ficheiros filter e form aha, sugeria pôr tudo num.

arieivs commented on app/views/organisations/partials/_filter_by_topic.html.erb:
```diff
@@ -1,5 +1,5 @@
-<%= simple_form_for :organisation_topic, url: organisations_path, method: :get do |f| %>
-  <%= f.input :topic, required: false, collection: Topic.distinct.pluck(:name), input_html: { class: "select2-topic", multiple: ""} %>
+<%= simple_form_for :organisation_topic, url: organisations_path, method: :get, remote: true do |f| %>
+  <%= f.input :topic, required: false, collection: Topic.distinct.pluck(:name), input_html: { class: "select2-topic", multiple: "multiple"} %>
```
o que é que o "multiple" muda? só por curiosidade

dmcarmo replied:
É para quando tivermos o ajax a funcionar.

dmcarmo committed:
Changes from pull request review

moved styles from _filter.scss to _form.scss and removed _filter.scss
renamed the simple_form class to w-48 and moved it to layout.scss
added the new class w-48 to the affected partials
